core/lib/mcxtx_api.h

```diff
@@ -135,7 +135,11 @@
      * all.  We do not need anything more than word alignment for OP_vldm/OP_vstm,
      * and dr_simd_t has no fields larger than 32 bits, so we have no padding.
      */
+#ifdef AARCH64
```

Comment:
style: indentation off -- but this should go away anyway (see other comment)

Reply:
Done.